samples/CommunityToolkit.Maui.Markup.Sample/Pages/SettingsPage.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Collections.Generic;
+using System.Globalization;
+using CommunityToolkit.Maui.Behaviors;
+using CommunityToolkit.Maui.Markup.Sample.Constants;
+using CommunityToolkit.Maui.Markup.Sample.Pages.Base;
+using CommunityToolkit.Maui.Markup.Sample.Services;
+using CommunityToolkit.Maui.Markup.Sample.ViewModels;
+using Microsoft.Maui;
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Graphics;
+using Microsoft.Maui.Layouts;
+
+namespace CommunityToolkit.Maui.Markup.Sample.Pages;
+
+class SettingsPage : BaseContentPage<SettingsViewModel>
+{
+	public SettingsPage(SettingsViewModel settingsViewModel) : base(settingsViewModel, "Settings")
+	{
+		Content = new AbsoluteLayout
+		{
+			Children =
+			{
+				new Label { Text = "Top Stories To Fetch", TextColor = ColorConstants.PrimaryTextColor, HorizontalTextAlignment = TextAlignment.Center }
+					.LayoutFlags(AbsoluteLayoutFlags.XProportional | AbsoluteLayoutFlags.WidthProportional)
+					.LayoutBounds(0,0,1,40),

```

Comment:
This can use `.TextCenter()` to replace `HorizontalTextAlignment = TextAlignment.Center`

```suggestion
				new Label { Text = "Top Stories To Fetch", TextColor = ColorConstants.PrimaryTextColor }
					.LayoutFlags(AbsoluteLayoutFlags.XProportional | AbsoluteLayoutFlags.WidthProportional)
					.LayoutBounds(0,0,1,40)
					.TextCenter(),

```

Reply:
Sorry I should have checked it also